### PR TITLE
Moved searchligh-ui to "Search" dashboard

### PR DESCRIPTION
**Pull request by tyr-hpe**

### CODE REVIEW

File: searchlight_ui/dashboards/search/dashboard.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2012 Nebula, Inc.
```
Comment (palecrow) [inline searchlight_ui/dashboards/search/dashboard.py]: I'd put the HPE copyright here

File: searchlight_ui/dashboards/search/search/urls.py [hunk 2]
```diff
@@ -15,10 +15,10 @@
 from django.conf.urls import patterns
 from django.conf.urls import url
 
-from searchlight_ui.dashboards.project.search import views
+from searchlight_ui.dashboards.search.search import views
```
Comment (palecrow) [inline searchlight_ui/dashboards/search/search/urls.py:18]: Do we need search/search?  or just search?

Comment (tyr-hpe) [inline searchlight_ui/dashboards/search/search/urls.py:18]: follows dashboard.panel pattern for now until we find a way to make panel display as dashboard.

File: searchlight_ui/dashboards/search/dashboard.py (hunk 1 above)
Comment (tyr-hpe) [inline searchlight_ui/dashboards/search/dashboard.py]: Fixed

**Pull request merged**